Automattic/jetpack
# Ensure the correct site goals option is returned

In our site base we had a getter for the `siteGoals` option which is not used or returned anywhere. This PR modifies this to be using the correct option which is `site_goals` and wires it to be returned with the site response.

Fixes #

## Proposed changes:

* Wire `site_goals` to be returned for a site

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Sync this PR to your sandbox
* Follow the onboarding flow at WordPress.com/start
* Once you arrive at the `goals` step select a few there
* Once you submit them check on your sandbox if they are properly set 
`wp option get site_goals --blog=BLOG_ID`
* Reload the page and inspect the site request. The `site_goals` should be a part of the response.
